dependency_injection/services.php

```diff
@@ -58,6 +58,7 @@
     $services->load('Glpi\Controller\\', $projectDir . '/src/Glpi/Controller');
     $services->load('Glpi\Http\\', $projectDir . '/src/Glpi/Http');
     $services->load('Glpi\DependencyInjection\\', $projectDir . '/src/Glpi/DependencyInjection');
+    $services->load('Glpi\Progress\\', $projectDir . '/src/Glpi/Progress');
```

Comment:
Is is alright to load the full `Progress` directory here ?

`SessionProgress` is not a service, shouldn't loading it in the dependency injection trigger an error as its constructor expect some parameters ?